libs/pages/application/src/lib/ui/page-settings-general/page-settings-general.spec.tsx

```diff
@@ -115,6 +115,9 @@ describe('PageSettingsGeneral', () => {
     const button = screen.getByTestId('submit-button')
 
     await userEvent.click(button)
-    expect(spy).toHaveBeenCalled()
+
+    waitFor(() => {
```

Comment:
Why is this needed now?

Reply:
I have the feeling because with set values for Repository and Branches after setting the Auth Providers, the DOM needed to wait for the Auth Providers values before validating others, that's why I added this